CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
+* [CHANGE] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
+* [FEATURE] Tracing(!): Added experimental Jaeger support #7148
+* [ENHANCEMENT] TSDB: Significantly reduce how much old WAL we keep around. #7098
```

Comment:
The "we" reads weirdly. How about: "TSDB: Significantly reduce WAL size kept around after truncation."